controller/src/test/java/io/pravega/controller/server/v1/ControllerServiceImplTest.java

```diff
@@ -303,33 +302,7 @@ public void createStreamTests() {
         status = result6.get();
         assertEquals(status.getStatus(), CreateStreamStatus.Status.SUCCESS);
     }
-
-    @Test
-    public void testCreateStreamThrowsLockFailed() {
```

Comment:
If you are removing this test case, then you should consider removing all other auxiliary code that came with the test case (follow `blockCriticalSection`).

Reply:
I have removed blockCriticalSection but retained the class TaskStoreFactoryForTests as that could be useful in authoring more tests in future.